src/Microsoft.OData.Core/ODataBatchReader.cs

```diff
@@ -771,7 +510,7 @@ private void IncreaseChangeSetSize()
         /// <summary>
         /// Resets the size of the current change set to 0.
         /// </summary>
-        private void ResetChangeSetSize()
+        internal void ResetChangeSetSize()

```

Comment:
>ResetChangeSetSize [](start = 22, length = 18)

Same as previous 2 comments; should be called in JSON format, consider making protected.